R/get_paleogeography.R

```diff
@@ -5,20 +5,21 @@
 #'   Global Plate Model.
 #'
 #' @param age \code{numeric}. Return a paleogeographic reconstruction based on
-#'   a numerical age in millions of years (between 0 and 550).
+#'   a numerical age in millions of years before present (between 0 and 550).
 #' @param interval_name \code{character}. Return a paleogeographic
 #'   reconstruction based on a specified time interval. Ignored if `age` is
 #'   supplied.
-#'
 #' @return An `sf` object containing the geometries of the paleogeographic map.
-#'
-#' @author Lewis A. Jones
-#'
+#' @section Developer(s):
+#'  Christopher D. Dean
+#' @section Reviewer(s):
+#'  Lewis A. Jones
```

Comment:
Looks like you got the developer and reviewer backwards

Reply:
Cheeky 🤣